new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -98,17 +98,19 @@
    * Check is a service exists in given namespace.
    *
    * @param serviceName the name of the service to check for
-   * @param label a Map of key value pairs the service is decorated with
-   * @param namespace in which the service is running
+   * @param label       a Map of key value pairs the service is decorated with
+   * @param namespace   in which the service is running
    * @return true if the service exists otherwise false
    * @throws ApiException when query fails
    */
-  public static boolean serviceReady(
+  public static Callable<Boolean> serviceReady(
```

Comment:
a service can't be "ready" - what are you really checking for? that it exists? that is has endpoints?

Reply:
I think its checking for existence. @sankarpn can you please confirm.

Reply:
The assertion only checks if a service exists. I guess @vanajamukkara is checking for whether the service exists, not ready.

Reply:
I will change the assertion method name to serviceExists to avoid confusion

Reply:
@markxnelson let me know if we should be checking for more than service existence.